navigator-menu/package.json

```diff
@@ -0,0 +1,36 @@
+{
+  "name": "navigator-menu",
+  "private": true,
+  "version": "0.0.0",
+  "type": "module",
+  "scripts": {
+    "dev": "vite",
+    "build": "vite build",
+    "preview": "vite preview"
```

Comment:
You can add the following script to help yourself :laughing: 
`    "eslint": "eslint -c eslint.config.cjs --ext .js,.jsx,.css,.cjs src/ --fix"`